### PR TITLE
patches: introduce patches directory

### PR DESCRIPTION
**Pull request by mardy**
Add a directory to hold patches needed in other layers/projects, along
with a readme explaining the rationale.

### CODE REVIEW

**Comment by breakreturn** [comment]
AFAIK we usually have patches that are temporary in a directory called patches-temporary or similar. @tobsan should know.

**Comment by mardy** [comment]
I updated the PR.

File: recipes-temporary-patches/bitbake/0001-bitbake-lib-bb-utils-fix-movefile-copy-to-dir-fallba.patch [hunk 1]
```diff
@@ -0,0 +1,64 @@
+From 685373a2d78af23de8e29bf4f771af6b7f28fdf0 Mon Sep 17 00:00:00 2001
+From: Alberto Mardegan <amardegan@luxoft.com>
+Date: Wed, 20 Sep 2017 16:35:00 +0200
+Subject: [PATCH] bitbake: lib/bb/utils: fix movefile() copy to dir fallback
```
Comment (GordanM) [inline recipes-temporary-patches/bitbake/0001-bitbake-lib-bb-utils-fix-movefile-copy-to-dir-fallba.patch]: Adding of this patch should be a separate commit IMO.

Comment (mardy) [reply recipes-temporary-patches/bitbake/0001-bitbake-lib-bb-utils-fix-movefile-copy-to-dir-fallba.patch]: Done
